test/reference.ts

```diff
@@ -0,0 +1,45 @@
+import * as msc from './msc_basis_transcoder/msc_basis_transcoder';
+
+/** Wrapper for MSC reference transcoder. */
+export class ReferenceTranscoder {
+  /**
+   * Matching transcode targets:
+   * - BC7_RGBA:       uastc_bc7.wasm
+   * - ASTC_4x4_RGBA:  uastc_astc.wasm
+   * - RGBA32:         uastc_rgba32_unorm.wasm
+   */
+  public TranscodeTarget: {[key: string]: unknown};
+
+  private basisModule;
+  private transcoder;
+  private imageInfo;
+
+  constructor (private readonly nBlocks: number = 8 * 1024 * 1024) {}
```

Comment:
The generator usage example uses `8 * 1024 * 1024` as a "more readable" way of expressing 2<sup>23</sup> blocks. This corresponds to, for example, 2048x4096 (8192x16384 px) or 1x8388608 (4x33554432 px) texture blocks.

To avoid potential misunderstanding, let's use something more canonical here.

Reply:
I'm not sure what you mean by more canonical, but I'll just omit the default value since it's passed into the constructor anyway.